### ISSUE TITLE
Request to enable 2 Greenshot Features - Full length webpage screenshots and save/load Greenshot Objects

### ISSUE THREAD

**Issue by jasondavis** [ShareX/ShareX#748]
Today I downloaded Greenshot just to see what it looked like by itself outside of ShareX and I discovered that it has 2 Features that I have always wanted ShareX to have!

Now that I see they exist in Greenshot already, I am pleading with you to please enable them in ShareX!

**1) Capture a Full length Webpage!**  

I have heard this request a few times over the past couple years and I believe the solution before and in the docs is to send users to download and use a Browser extension instead!  

This is built in part of Greenshot!  Isn't there a way to enable it in ShareX?

![0pjxwzl](http://i.imgur.com/lRUpUjt.png)

**2) Save and Load Objects in Greenshot Editor**    

As a longtime user of SHareX and using the app all day everyday, I was very excited to see this feature in Greenshot.  That is if we can get it exposed and working in ShareX too!  

So usage example:  
Lets say you are taking multiple screenshots of things and you have a standard set of text or number steps you add to a certain type of screenshot.  THis allows you to go to the "Object" menu and click export objects and it will save all objects you have added to the screenshot.

Now at a later time when you are editing a screenshot, you can go to the Object menu again and this time load in your objects file.  It will then insert all the objects you had made into your new screenshot image!  This could be text, shapes, arrows, lines, everything you can do in the editor is an object and you have the ability to export and save a set of object and then import them into any other screenshot you make in the future!

This is a super useful feature for myself and is available in Greenshot already!

![aukcj3f](https://cloud.githubusercontent.com/assets/186818/8369581/17c67354-1b8a-11e5-9e80-0f861f33c838.png)

**I am a long time user of ShareX and I greatly appreciate all the time put into and sharing with the world, truly thankful as I use this app more than any other program!  I use it all day, everyday.  I ask that you please consider my requests above, thank you**

**- Jason**

**Comment by Jaex** [comment]
1) I was added web capture using IE long time ago in ZScreen but it was not good enough for my quality standards therefore I was not added it to ShareX. Like it was not giving expected result in some web sites.
And about IE window list, who using IE in this century? It is completely useless.

2) I was removed it because I thought nearly nobody would use it while trying to simplify UI and remove not used codes. Now I can't find codes of it and gonna be difficult to implement because removed many unused class files etc.

**Comment by Jaex** [comment]
I found codes in ZScreen project.
Maybe I should add it even if it is very bad in my opinion to use IE to render in background.

**Comment by Jaex** [comment]
![](https://dl.dropboxusercontent.com/u/14076298/ShareX/2015/06/ShareX_MB1iplYPYq.png)

**Comment by jasondavis** [comment]
Sorry just now saw your replies.  While I agree no self respecting developer uses IE, it does generally exist on a PC for at least testing and I figured it was better than requiring the use of a Chrome extension separate from the program.  At least in some cases!

Anyways it seems you have added in support for it now so thank you that is great!

Here is a demo screenshot that turned out amazing!  http://i.imgur.com/89GzOyi.png

This website has a fixed header bar and a fixed sidebar advertisement block which generally show chopped up multiple times in the screenshot when using Chrome screenshot extensions...using the took you implemented, it shows them perfectly better than any other screenshot tool I have used on that site so that is very awesome!
